OmiseSDK/PaymentChooserViewController.swift

```diff
@@ -197,6 +203,9 @@ class PaymentChooserViewController: AdaptableStaticTableViewController<PaymentCh
             }
         case ("GoToTrueMoneyFormSegue"?, let controller as TrueMoneyFormViewController):
             controller.flowSession = self.flowSession
+        case ("GoToFPXFormSegue"?, let controller as FPXFormViewController):
+            controller.showingValues = capability?[.fpx]?.banks ?? []
```

Comment:
For other payment methods the merchants don't have to use the Capability API, they can specify the payment methods manually. For FPX it's mandatory to use the Capability API?

And should we show something when there are no payment methods available?
![Simulator Screen Shot - iPhone 12 - 2021-03-02 at 15 37 22](https://user-images.githubusercontent.com/712728/109621333-6c168900-7b6d-11eb-89f2-d4c6d847174f.png)

Reply:
> For FPX it's mandatory to use the Capability API?

Yeah, merchants should use the capability API to get the latest list of banks with their online/offline status provided through the Capability API

Reply:
I'm not sure what a valid fallback would be as we would have to guess at all the banks being online.

Reply:
Okay I see. In this case it probably makes no sense to provide default banks. But I think it would be good to display a message on this white screen, something like: `FPX payment is currently unavailable.`
And in the `PaymentChooserViewController` we could also check if the Capabilities contain FPX payment and if banks are available. If not we could remove FPX from the list of available methods.
@danfowler What do you think?